File: docs/scripts/check_table_structure.py
import pyodbc

conn = pyodbc.connect(
    "DRIVER={ODBC Driver 17 for SQL Server};"
    "SERVER=localhost;"
    "DATABASE=klas;"
    "Trusted_Connection=yes;"
)

cursor = conn.cursor()

# Get table structure
cursor.execute("""
SELECT 
    COLUMN_NAME, 
    DATA_TYPE, 
    IS_NULLABLE,
    COLUMN_DEFAULT
FROM INFORMATION_SCHEMA.COLUMNS 
WHERE TABLE_NAME = 'Rack_Shelf_Labels'
ORDER BY ORDINAL_POSITION
""")

print("Rack_Shelf_Labels table structure:")
for row in cursor.fetchall():
    print(f"  {row[0]}: {row[1]} (nullable: {row[2]}, default: {row[3]})")

# Get a sample row
cursor.execute("SELECT TOP 1 * FROM Rack_Shelf_Labels")
sample = cursor.fetchone()
if sample:
    print(f"\nSample row: {sample}")
else:
    print("\nNo existing rows found")

conn.close()

File: docs/scripts/check_table_structure.php
<?php

require_once __DIR__ . '/vendor/autoload.php';

$app = require_once __DIR__ . '/bootstrap/app.php';
$kernel = $app->make(Illuminate\Contracts\Console\Kernel::class);
$kernel->bootstrap();

echo "=== MOTHER_APPLICATIONS TABLE STRUCTURE ===\n\n";

try {
    $columns = DB::connection('sqlsrv')->select("
        SELECT COLUMN_NAME, DATA_TYPE, IS_NULLABLE, CHARACTER_MAXIMUM_LENGTH 
        FROM INFORMATION_SCHEMA.COLUMNS 
        WHERE TABLE_NAME = 'mother_applications' 
        ORDER BY ORDINAL_POSITION
    ");
    
    foreach ($columns as $col) {
        $length = $col->CHARACTER_MAXIMUM_LENGTH ? "({$col->CHARACTER_MAXIMUM_LENGTH})" : '';
        echo "{$col->COLUMN_NAME} {$col->DATA_TYPE}{$length} (nullable: {$col->IS_NULLABLE})\n";
    }
    
    echo "\n=== Sample Records ===\n";
    $records = DB::connection('sqlsrv')->table('mother_applications')
        ->limit(2)
        ->get();
    
    if ($records->count() > 0) {
        foreach ($records as $record) {
            echo "ID: {$record->id}, NP: " . ($record->np_fileno ?? 'NULL') . ", File: " . ($record->fileno ?? 'NULL') . ", Applicant: " . ($record->applicant_type ?? 'NULL') . "\n";
        }
    } else {
        echo "No records found.\n";
    }
    
} catch (Exception $e) {
    echo "Error: " . $e->getMessage() . "\n";
}